frontend/apps/reader/modules/readerpaging.lua

```diff
@@ -159,7 +167,11 @@ function ReaderPaging:setupTouchZones()
             screen_zone = {
                 ratio_x = 0, ratio_y = 0, ratio_w = 1, ratio_h = 1,
             },
-            handler = function(ges) return self:onSwipe(nil, ges) end
+            handler = function(ges)
+                if G_reader_settings:readSetting("page_turns") ~= "tap" then
+                    return self:onSwipe(nil, ges)
+                end
+            end
```

Comment:
While I like the idea of nipping this directly in the registerTouchZone closure, this is problematic for the swipe handler, because it doesn't handle *only* page turns.

It's also responsible for the refresh swipe, plus the weird flip mode thingy in PDFs.

Reply:
I'm afraid I don't know how to solve that conflict. It sounds like the solution would be something way beyond my current knowledge. Any ideas?

Reply:
Move the check to the handler itself, and only trap west & east gestures?

Reply:
I keep getting crashes when I move the check and I can't figure out how to trap only west and east gestures.

Reply:
I pushed some commits in your branch that should do this ;).

Reply:
That looks a lot better than what I could have done on my own. Thanks a lot!